src/pages/Account/Tabs/ModulesTab/Tabs.tsx

```diff
@@ -93,7 +93,7 @@ function ModulesTabs({address}: {address: string}): JSX.Element {
         break;
     }
     eventName && logEvent(eventName);
-  }, [value]);
+  }, [value, logEvent]);
```

Comment:
looking at the implementation of useLogEvent it looks like it returns a new function on every rerender and hence would retrigger this effect every time, which is probably not what we want.